### ISSUE TITLE
DICOM-Object/Presentation state toolbar

### ISSUE THREAD

**Issue by chess42** [nroduit/Weasis#377]
Firstly, thank you so much for your excellent work.

Is it possible to implement a toolbar of sorts to navigate between annotations/imported DICOM-objects? At the moment, objects in the presentation state must be manually searched for and "stumbled upon" via viewing/scrolling through every slice.

A possible solution would be a "Presentation" rider under "Draw & Measure" listing:
- name of DICOM object
[- slice number]
- button for jumping to that slice
- possibly a toggle button to turn it on or off

Thank you for your consideration.

Sincerely,
Christopher

**Comment by nroduit** [comment]
Yes, there are several features already on the list of things to do to improve the management of Presentation State.

Currently, some major features with a high priority must be implemented and the resources on this project are very limited. We need contributors for this kind of feature.
